apps/system/style/chrome/chrome.css

```diff
@@ -243,7 +248,8 @@
 
 
 /* Private browser icon in rocketbar */
-.appWindow.private > .chrome.maximized .title {
+.appWindow.private > .chrome-plain.maximized .title,
```

Comment:
Does having this not cause the padding to be double? Before this only was applied to ```.title```, and now it's also applied to ```.chrome-title-container```